src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Usage/DetectPLINQNopsTest.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.NetCore.Analyzers.Usage;
+using Xunit;
+
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Usage.DetectPLINQNopsAnalyzer,
+    Microsoft.NetCore.Analyzers.Usage.DetectPLINQNopsFixer>;
+
+namespace Microsoft.CodeAnalysis.NetAnalyzers.UnitTests.Microsoft.NetCore.Analyzers.Usage
+{
+    public sealed class DetectPLINQNopsTest
+    {
+        [Fact]
+        public async Task AsParallelToListInForeach_SingleDiagnostic()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+    using System;
```

Comment:
Could you remove all tabs to have something more consistent with the other tests in the repo.